tool/tctl/common/token_command.go

```diff
@@ -257,11 +257,13 @@ func (c *TokensCommand) Add(ctx context.Context, client auth.ClientI) error {
 		if len(proxies) == 0 {
 			return trace.NotFound("cluster has no proxies")
 		}
+		setRoles := strings.ToLower(strings.Join(roles.StringSlice(), "\\,"))
```

Comment:
Should we enable `app & discovery` if type = kube?

Reply:
It would be impossible for user to add token just for `kube` roles then, do you think this is ok?

Reply:
It's the same behavior we have in adding new resources via web UI.

Reply:
Added in https://github.com/gravitational/teleport/pull/30291/commits/e3ce9ac0c6b74ed80f7e9710f5f2f6d105f9b46a